src/test/ui/pattern/const-pat-ice.stderr

```diff
@@ -11,3 +11,8 @@ note: rustc VERSION running on TARGET
 
 note: compiler flags: FLAGS
 
+query stack during panic:
+we're just showing a limited slice of the query stack
+#0 [check_match] match-checking `main`
+#1 [analysis] running analysis passes on this crate
+end of query stack
```

Comment:
I think this shouldn't be shown when only showing some frames.

Reply:
ok I will fix, thanks

Reply:
@bjorn3 sorry can you explain that which part I should change?
I think it is correct, because RUST_BACKTRACE is 0 here, so then we will show ```we're just showing a limited slice of the query stack``` message

Reply:
What they are suggesting is to remove the `end of query stack` message. Maybe either print `end of query stack` or `query stack printing limited, use RUST_BACKTRACE=1 to see more` depending on whether we're limiting or not? So instead of printing `we're just showing a limited...` before the stack, we show that after the stack at the cutoff point where it normally says `end of query stack`

Reply:
aa ok gotcha thanks